phpstan-baseline.neon

```diff
@@ -0,0 +1,22 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Strict comparison using \\=\\=\\= between null and App\\\\Entity\\\\User will always evaluate to false\\.$#"
```

Comment:
Instead of putting that in the baseline, the right solution is to fix the invalid phpdoc that says `/** @var User $user */` in DeleteUserCommand when it should say `/** @var User|null $user */`

Reply:
@stof Sure, as written in the description, I wasn't sure if fixing the reported errors should be part of this PR. I fixed the type hint in 39c3d07e9ee90c8a49dcff96f949a4ab36241124.

Should I try fixing some more of the remaining issues as well in this PR?